.github/workflows/release-prepare.yml

```diff
@@ -195,18 +193,6 @@ jobs:
         if: ${{ failure() }}
         run: sudo dmesg
 
-      - name: Cache cargo related build files
```

Comment:
No need to save and restore separately, the action does that for us

Reply:
For the client repos I've split these actions into restore and save, because when the tests fail, nothing is saved. Does this make sense here too?

Reply:
It'd be justifiable but this isn't a long pipeline, it's just two steps and only one of them runs cargo commands - unlike tests which can fail regularly, we don't really expect anything in this pipeline to fail often. I think it's okay like this